Main.java

```diff
@@ -9,7 +11,46 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Create a Scanner for user input 
+     Scanner input = new Scanner(System.in); 
+
+    // prompt the user to enter the size of their pizza 
+    System.out.println("What is the size of the pizza (in cm):"); 
+
+    // create a variable for pizza size 
+    double Pizza_Size = input.nextInt();  
+
+    // create variables for the charges on the pizza 
+    double labour_ch = 0.75;
+    double oven_ch = 0.99;
+    double per_cm_ch = Pizza_Size*0.50;  
+
+    // create variable for the Subtotal,taxes, and total 
+    double Sub_total = per_cm_ch + oven_ch + labour_ch; 
+    double tax_total = (Sub_total/100)*13;
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. This means always start with a lowercase letter and no underscores (_) between words.